src/PowerShell/Utilities/Utils.ts

```diff
@@ -0,0 +1,55 @@
+import * as os from 'os';
+import * as exec from '@actions/exec';
+import * as io from '@actions/io';
+
+import Constants from '../Constants';
+import PowerShellToolRunner from './PowerShellToolRunner';
+
+export default class Utils {
+    static async getLatestModule(moduleName: string): Promise<string> {
+        let output: string = "";
+        let error: string = "";
+        const options: any = {
+            listeners: {
+                stdout: (data: Buffer) => {
+                    output += data.toString();
+                },
+                stderr: (data: Buffer) => {
+                    error += data.toString();
+                }
+            }
+        };
+        await PowerShellToolRunner.init();
+        await PowerShellToolRunner.executePowerShellCommand(`(Get-Module -Name ${moduleName} -ListAvailable | Sort-Object Version -Descending | Select-Object -First 1).Version.ToString()`, options);
```

Comment:
As discussed offline with @arjgupta returning output as a structured object makes sense. Error you may handle as you wish.